app/controllers/users_controller.rb

```diff
@@ -12,8 +22,30 @@ def create
     end
   end
 
+  def update
+    if @user&.update(user_params)
+      redirect_to @user, notice: t("users.update.success")
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    if @user&.destroy
+      redirect_to users_path, notice: t("users.destroy.success")
+    else
+      redirect_to users_path, alert: t("users.destroy.failed")
+    end
+  end
+
   private
 
+  def set_user
+    @user = User.find_by(id: params[:id])
+    return if @user
+    redirect_to users_path, alert: t("users.not_found")
```

Comment:
```suggestion
    return if @user
    
    redirect_to users_path, alert: t("users.not_found")
```
Thêm 1 line trống dưới return if